### DISCUSSION TITLE
Moby release / updating dependency github.com/moby/swarmkit/v2

### DISCUSSION THREAD

**Discussion by cf-sewe** [moby/moby#46597] [Q&A] [answered]
I am currently waiting for an update of swarmkit, to address an AppArmor issue for our Swarm services. 
It seems there is now an [feature](https://github.com/moby/swarmkit/pull/3149) that would help, but when will it be available in an official release?

**Comment by neersighted** [comment, marked as answer]
This shipped in the v25 release.
